File: backend/src/main/java/space/davids_digital/vk_gpt_bot/orm/entity/AppCeoEntity.java
package space.davids_digital.vk_gpt_bot.orm.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "app_ceos")
public class AppCeoEntity {
    @Id
    @Column(name = "user_id")
    public long userId;
}

File: backend/src/main/java/space/davids_digital/sweetie/orm/entity/AppCeoEntity.kt
package space.davids_digital.sweetie.orm.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "app_ceos")
class AppCeoEntity {
    @Id
    @Column(name = "user_id")
    var userId: Long = 0
}